mws/utils/params.py

```diff
@@ -1,14 +1,15 @@
 """Parameter manipulation utilities."""
 
-from collections.abc import Iterable, Mapping
+from collections.abc import Mapping
+from typing import Any, Iterable, Optional, Union
 from urllib.parse import quote
 import datetime
 import json
 
 from mws.errors import MWSError
 
 
-def enumerate_param(param, values):
+def enumerate_param(param: str, values: Union[list, set, tuple]) -> dict:
```

Comment:
Spent some time adding type annotations to a lot of different methods here. Honestly, I think I was bored that day.